docs/source/app-dev/bindings-java/quickstart.rst

```diff
@@ -356,124 +356,126 @@ The *Issuers* of the two Ious, which are involved in the transaction because the
 
 For a deeper introduction to Daml, consult the :doc:`Daml Reference </daml/reference/index>`.
 
-.. _quickstart-scenarios:
+.. _quickstart-scripts:
 
-Test using scenarios
+Test using Daml Script
 ====================
 
-You can check the correct authorization and privacy of a contract model using *scenarios*: tests that are written in Daml.
+You can check the correct authorization and privacy of a contract model using *scripts*: tests that are written in Daml.
 
-Scenarios are a linear sequence of transactions that is evaluated using the same consistency, conformance and authorization rules as it would be on the full ledger server or the sandbox ledger. They are integrated into Daml Studio, which can show you the resulting transaction graph, making them a powerful tool to test and troubleshoot the contract model.
+Scripts are a linear sequence of transactions that is evaluated using the same consistency, conformance and authorization rules as it would be on the full ledger server or the sandbox ledger. They are integrated into Daml Studio, which can show you the resulting transaction graph, making them a powerful tool to test and troubleshoot the contract model.
 
-To take a look at the scenarios in the quickstart application, open ``daml/Tests/Trade.daml`` in Daml Studio.
+To take a look at the scripts in the quickstart application, open ``daml/Tests/Trade.daml`` in Daml Studio.
 
-A scenario test is defined with ``trade_test = scenario do``. The ``submit`` function takes a submitting party and a transaction, which is specified the same way as in contract choices.
+A script test is defined with ``trade_test = script do``. The ``submit`` function takes a submitting party and a transaction, which is specified the same way as in contract choices.
 
 The following block, for example, issues an ``Iou`` and transfers it to Alice:
 
 .. literalinclude:: quickstart/template-root/daml/Tests/Trade.daml
   :language: daml
-  :start-after: -- BEGIN_SCENARIO
-  :end-before: -- END_SCENARIO
+  :start-after: -- BEGIN_SCRIPT
+  :end-before: -- END_SCRIPT
 
-Compare the scenario with the ``setup`` scenario in ``daml/Main.daml``. You will see that the scenario you used to initialize the sandbox is an initial segment of the ``trade_test`` scenario. The latter adds transactions to perform the trade you performed through Navigator, and a couple of transactions in which expectations are verified.
+Compare the script with the ``initialize`` script in ``daml/Main.daml``. You will see that the script you used to initialize the sandbox is an initial segment of the ``trade_test`` script. The latter adds transactions to perform the trade you performed through Navigator, and a couple of transactions in which expectations are verified.
```

Comment:
I updated both `daml/Main.daml` and `daml/Tests/Trade.daml` so 

> the script you used to initialize the sandbox is an initial segment of the ``trade_test`` script

is true now.